test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -439,10 +452,17 @@ end
 function source(::EntrDetr{up_ρaq_tot{i}}, atmos, args) where {i}
     @unpack E_dyn, Δ_dyn, E_trb, env, ρa_up, ts_en = args.precomputed.turbconv
     up = args.state.turbconv.updraft
-    q_tot_en = total_specific_humidity(ts_en)
-    entr = fix_void_up(ρa_up[i], (E_dyn[i] + E_trb[i]) * q_tot_en)
-    detr =
-        fix_void_up(ρa_up[i], (Δ_dyn[i] + E_trb[i]) * up[i].ρaq_tot / ρa_up[i])
+    if atmos.moisture isa DryModel
+        entr = FT(0)
+        detr = FT(0)
```

Comment:
Why are entrainment and detrainment zero in a DryModel ?

Reply:
this is in the computation of entr and detr sources for qt

Reply:
good point, thanks!